### PR TITLE
Pull Request with Current Website Code

### PR DESCRIPTION
**Pull request by OriginalName2**
Making a pull request with current website code finished. Planning to add a lot more information to contacts and team/robot once I get more info on the topic. Will add more images later on when I get them.

### CODE REVIEW

**Review by abrightwell — CHANGES_REQUESTED** [review]
Some items that need to be addressed:

1). Move the CSS into a seperate `.css` file and link to it appropriately. Therefore you are not copy pasting and changes to style only need to happen in one place. 

2). Give each file an appropriate file extension.

**Comment by OriginalName2** [comment]
I've made requested changes with the files in the repository, 19 commits

File: contacts.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+<link href='http://fonts.googleapis.com/css?family=Lato&subset=latin,latin-ext' rel='stylesheet' type='text/css'>
+<link href="style.css" rel="stylesheet">
+
```
Comment (abrightwell) [inline contacts.html]: This should be under `<header>`. Fix throughout.

**Review by abrightwell — CHANGES_REQUESTED** [review]
It's getting there, but need to fix the location of the stylesheet links as well, let's format the files with appropriate spacing and indentation.

**Pull request closed without merging**